### PR TITLE
lectia-2-2: add new components from lesson

### PR DESCRIPTION
**Pull request by alexcracea**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,6 +29,27 @@ <h2 class="section__title">Background Box</h2>
...
+        <div class="section__content">
+          <div class="background-box"></div>
+          // codul la tema despre background-box
+        </div>
+      </div>
+      <div class="section" id="background-box">
+        <h2 class="section__title">Background Box</h2>
+        <div class="section__content">
+          <div class="background-box"></div>
+          // codul la tema despre background-box
```
Comment (CiochinaAndrei) [inline index.html:43]: Aici treb de cumpărat un Kebab

**Review by alexanderfilip1 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -29,6 +29,27 @@ <h2 class="section__title">Background Box</h2>
...
       </div>
+      <div class="section" id="background-box">
+        <h2 class="section__title">Background Box</h2>
+        <div class="section__content">
+          <div class="background-box"></div>
+          // codul la tema despre background-box
+        </div>
+      </div>
+      <div class="section" id="background-box">
+        <h2 class="section__title">Background Box</h2>
```
Comment (macovei1337) [inline index.html:40]: New section name

File: scss/main.scss [hunk 3]
```diff
@@ -4,6 +4,8 @@
 @import 'base/mixins';
 @import 'components/backround-box';
 @import 'components/fonts';
+@import 'components/basic-box';
+@import 'components/people-layout';
```
Comment (CiochinaAndrei) [inline scss/main.scss]:
```suggestion
pune ceva aici
```

**Review by CiochinaAndrei — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (alexcracea) [reply index.html:43]: done

File: scss/main.scss [hunk 4]
```diff
@@ -4,6 +4,8 @@
 @import 'base/mixins';
 @import 'components/backround-box';
 @import 'components/fonts';
```
Comment (Alexandru077) [inline scss/main.scss]: please delete this line

**Review by Alexandru077 — CHANGES_REQUESTED** [review]

**Review by macovei1337 — CHANGES_REQUESTED** [review]

File: scss/main.scss (hunk 4 above)
Comment (alexcracea) [reply scss/main.scss]: done

File: index.html (hunk 2 above)
Comment (alexcracea) [reply index.html:40]: @macovei1337 what should be the new name?

File: scss/main.scss [hunk 5]
```diff
@@ -16,3 +18,11 @@
 aside {
   @include fix-inline-blocks();
 }
+
+footer {
+  @include fix-inline-blocks(30px);
```
Comment (Ciosa007) [inline scss/main.scss:22]: da "@include"  noi am trecut la lectia 1 ?

**Review by Ciosa007 — APPROVED** [review]

File: scss/main.scss (hunk 5 above)
Comment (alexcracea) [reply scss/main.scss:22]: da, indeed am trecut

**Pull request merged**